types/node/index.d.ts

```diff
@@ -419,11 +419,11 @@ declare var Buffer: {
      * @param totalLength Total length of the buffers when concatenated.
      *   If totalLength is not provided, it is read from the buffers in the list. However, this adds an additional loop to the function, so it is faster to provide the length explicitly.
      */
-    concat(list: Buffer[], totalLength?: number): Buffer;
+    concat(list: Buffer[] | Uint8Array[], totalLength?: number): Buffer;
```

Comment:
`Buffer` inherits from `Uint8Array` since Node version 3.0 (https://nodejs.org/api/buffer.html#buffer_buffers_and_typedarray), so

- That change should actually go to all versions down to 3.0
- We could just use `Uint8Array` as the argument type

Or am I mistaken?

Reply:
Yeah, sounds like the appropriate change is to use `Uint8Array` unless I'm missing something.